packages/sidenav/README.md

```diff
@@ -0,0 +1,264 @@
+![Farmblocks logo](https://user-images.githubusercontent.com/7760/31051341-4d280118-a63c-11e7-9e8f-3b375ca8f9a0.png)
+
+# Farmblocks SideNav <!-- omit in toc -->
+
+A React Sidebar Navigation component
+
+## Table of Contents <!-- omit in toc -->
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [API](#api)
+  - [SideNav](#sidenav)
+  - [NavItem](#navitem)
+- [Helpers](#helpers)
+  - [PageWrapper](#pagewrapper)
+- [License](#license)
+
+## Installation
+
+```sh
+npm install @crave/farmblocks-sidenav
+```
+
+or with yarn
+
+```sh
+yarn add @crave/farmblocks-sidenav
+```
+
+## Usage
+
+Basic example
+
+```jsx
+import React from "react";
+import { render } from "react-dom";
+import SideNav from "@crave/farmblocks-sidenav";
+
+const App = () => <SideNav render={() => <div>Sidebar Content</div>} />;
+
+render(<App />, document.getElementById("root"));
+```
+
+Basic example with NavItem and PageWrapper
+
+```jsx
+import React from "react";
+import { render } from "react-dom";
+import SideNav, { NavItem, PageWrapper } from "@crave/farmblocks-sidenav";
+
+const App = () => (
+  <>
+    <SideNav
+      render={() => (
+        <>
+          <NavItem active>Item 1 (active)</NavItem>
+          <NavItem>Item 2</NavItem>
+          <NavItem>Item 3</NavItem>
+        </>
+      )}
+    />
+    <PageWrapper expanded>
+      <h1>Page Content</h1>
+    </PageWrapper>
+  </>
+);
+
+render(<App />, document.getElementById("root"));
+```
+
+Complete Example
+
+```jsx
+import React from "react";
+import { render } from "react-dom";
+import SideNav, { NavItem, PageWrapper } from "@crave/farmblocks-sidenav";
+
+const App = () => {
+  const [expanded, { toggle, collapse }] = useToggle(false);
+  const [selected, setSelected] = useState(tabs[0]);
+  const tabs = ["purveyor", "order", "search", "meat"];
+
+  return (
+    <>
+      <SideNav
+        expanded={expanded}
+        onToggle={toggle}
+        onClose={collapse}
+        render={props => (
+          <>
+            <NavHeader />
+            {tabs.map(tab => (
+              <NavItem
+                key={tab}
+                onClick={() => setSelected(tab)}
+                active={tab === selected}
+                icon={`wg-${tab}`}
+                {...props}
+              >
+                {tab}
+              </NavItem>
+            ))}
+          </>
+        )}
+      />
+
+      <PageWrapper expanded={expanded}>
+        <LoremIpsumBlock />
+      </PageWrapper>
+    </>
+  );
+};
+
+render(<App />, document.getElementById("root"));
+```
+
+To see these and more examples running, visit our [storybook](https://cravefood.github.io/farmblocks/?path=/story/sidenav-sidenav--side-nav-overlay)
+
+## API
+
+### SideNav
+
+- **expanded** (_Boolean_) = `true`
```

Comment:
The props documentation is duplicated now. We can remove this and link it to the storybook docs page.